src/mpyl/project.py

```diff
@@ -336,6 +340,9 @@ def from_config(values: dict):
             command=TargetProperty.from_config(values.get("command", {})),
             args=TargetProperty.from_config(values.get("args", {})),
             labels=list(map(KeyValueProperty.from_config, values.get("labels", []))),
+            strategy_type=deployment_strategy.get("strategyType", "RollingUpdate"),
+            max_surge=deployment_strategy.get("maxSurge", "25%"),
+            max_unavailable=deployment_strategy.get("maxUnavailable", "25%"),
```

Comment:
I would put the default values in the mpyl_config itself (you can see there are already default values encoded there). Because right now the defaults are still hard coded in this section, but it might be nice to be able to configure those as well. Now that we're changing this anyway :)

Reply:
For example take a look at the following dataclass that parses the default values for the resources from the mpyl config `mpyl.steps.deploy.k8s.chart.ResourceDefaults`

Reply:
Addressed in https://github.com/Vandebron/mpyl/pull/356/commits/69e15036831fbe7c27b8f0e4f1c03a77a26522a4